vignettes/RPyGeo.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "RPyGeo"
 author: "Marc Becker and Fabian Polakowski"
-date: "23.10.2018"
+date: "`r Sys.Date()`"
```

Comment:
Done

Reply:
What do you mean with Done? This will be included into the master as soon as you accept the PR into the master branch. But pls still wait with merging since I will add a few other commits.